### DISCUSSION TITLE
What version of PHP is supported?

### DISCUSSION THREAD

**Discussion by tooliedotter** [YOURLS/YOURLS#3247] [Q&A] [answered]
Hi, I'm about to move my YOURLS website to a new server with PHP 8.0.x. Will your software work on that version of PHP? I would hate to lose my link shortener! Thanks.

**Comment by dgw** [comment, marked as answer]
https://github.com/YOURLS/YOURLS/wiki/Requirements

**Reply by tooliedotter** [reply]
OK good to know. Since the 1.8.2 upgrade my software not working, but I'll try it on the new server and let you know. Thank you!

**Reply by tooliedotter** [reply]
I did get it working, thank you!
